xml/Sector.xml

```diff
@@ -1,3354 +1,3354 @@
-			</description>
-			<category>151</category>
-		</codelist-item>
-		<codelist-item status="withdrawn">
-			<code>15161</code>
-			<name>
-				<narrative>Elections</narrative>
-			</name>
-			<description>
-				<narrative>Electoral assistance and monitoring, voters' education [other than in connection with UN peace building (15230)].</narrative>
-			</description>
-			<category>151</category>
-		</codelist-item>
-		<codelist-item status="withdrawn">
-			<code>15162</code>
-			<name>
-				<narrative>Human rights</narrative>
-			</name>
-			<description>
-				<narrative>Monitoring of human rights performance; support for national and regional human rights bodies; protection of ethnic, religious and cultural minorities [other than in connection with un peace building (15230)].</narrative>
-			</description>
-			<category>151</category>
-		</codelist-item>
-		<codelist-item status="withdrawn">
-			<code>15163</code>
-			<name>
-				<narrative>Free flow of information</narrative>
-			</name>
-			<description>
-				<narrative>Uncensored flow of information on public issues, including activities that increase the professionalism, skills and integrity of the print and broadcast media (e.g. training of journalists).</narrative>
-			</description>
-			<category>151</category>
-		</codelist-item>
-		<codelist-item status="withdrawn">
-			<code>15164</code>
-			<name>
-				<narrative>Women's equality organisations and institutions</narrative>
-			</name>
-			<description>
-				<narrative>Support for institutions and organisations (governmental and non-governmental) working for gender equality and women's empowerment.</narrative>
-			</description>
-			<category>151</category>
-		</codelist-item>
-		<codelist-item>
-			<code>15170</code>
-			<name>
-				<narrative>Women’s equality organisations and institutions</narrative>
-				<narrative xml:lang="fr">Organisations et institutions pour la lutte contre la corruption</narrative>
-			</name>
-			<description>
-				<narrative>Support for institutions and organisations (governmental and non-governmental) working for gender equality and women’s empowerment.</narrative>
-				<narrative xml:lang="fr">Soutien aux institutions et organisations (gouvernementales et non gouvernementales) qui œuvrent pour l’égalité homme-femme et l’autonomisation des femmes.</narra
```

Comment:
fixes erroneous code (`23065`); introduces typo (“barrag es” should be “barrages”)